uk.co.compucorp.civicrm.hrleaveandabsences/views/manager-leave/components/manager-leave-calendar.html

```diff
@@ -1 +1,798 @@
-<div>Calender</div>
+<section>
+  <div class="panel panel-default">
+
+    <div class="panel-heading chr_manager_calendar__header">
+      <div>
+        <ui-select ng-change="ctrl.refresh()"
+                   ng-model="ctrl.filters.leaveRequest.contact">
+          <ui-select-match placeholder="Staff Member" allow-clear>
+            <span ng-bind="$select.selected.display_name"></span>
+          </ui-select-match>
+          <ui-select-choices repeat="user in (ctrl.filteredUsers | filter: $select.search) track by user.id">
+            <span ng-bind="user.display_name"></span>
+          </ui-select-choices>
+        </ui-select>
+      </div>
+      <div class="chr_custom-select">
+        <select
+          ng-options="region.label for region in ctrl.regions track by region.id"
+          ng-model="ctrl.filters.contact.region"
+          ng-change="ctrl.refresh(region)">
+          <option value="">Region</option>
+        </select>
+      </div>
+      <div class="chr_custom-select">
+        <select
+          ng-options="department.label for department in ctrl.departments track by department.id"
+          ng-model="ctrl.filters.contact.department"
+          ng-change="ctrl.refresh()">
+          <option value="">Department</option>
+        </select>
+      </div>
+      <div class="chr_custom-select">
+        <select
+          ng-options="levelType.value for levelType in ctrl.levelTypes track by levelType.id"
+          ng-model="ctrl.filters.contact.level_type"
+          ng-change="ctrl.refresh()">
+          <option value="">Level Type</option>
+        </select>
+      </div>
+      <div class="chr_custom-select">
+        <select
+          ng-options="location.label for location in ctrl.locations track by location.id"
+          ng-model="ctrl.filters.contact.location"
+          ng-change="ctrl.refresh()">
+          <option value="">Locations</option>
+        </select>
+      </div>
+      <div class="checkbox">
+        <label>
+          <input type="checkbox"
+                 ng-change = "ctrl.refresh()"
+                 ng-model="ctrl.filters.leaveRequest.pending_requests"> Show only those who are taking leave
+        </label>
+      </div>
+    </div>
+    <div class="panel-subheading">
+      <div class="row">
+        <div class="col-sm-5">
+          <ui-select multiple
+                     limit="4"
```

Comment:
As this is a static HTML+CSS ticket not implementing it now, will do as part of next ticket.